### ISSUE TITLE
[CW-1981] Making ChatWoot more accessible a a Vultr One click Installation

### ISSUE THREAD

**Issue by sobreiti** [chatwoot/chatwoot#6894]
A lot of chatWoot users including me have move to vultr from DO and was wondering if we could get the same accessbility and ease in terms of one-click installation as was on DO. 

This would not only help make chatwoot more accessible but easier for people moving to Vultr and trying to set up an instance.

<sub>[CW-1981](https://linear.app/chatwoot/issue/CW-1981/making-chatwoot-more-accessible-a-a-vultr-one-click-installation)</sub>

**Comment by kalebwd** [comment]
I have also requested this and was given a flat "no"

**Comment by iammattmartin** [comment]
Shouldn't this be more aimed at the Vultr team than CW?

**Comment by kalebwd** [comment]
@iammattmartin  No, just as Heroku didn't create theirs.

**Comment by kalebwd** [comment]
@iammattmartin No. https://www.vultr.com/marketplace/become-a-verified-vendor/

**Comment by movilreup** [comment]
Other great options and easy to install manage including backup mail server and other is via cloudron. In new fresh vultr instance install cloudron and into the available app from cloudron install and configure chatwoot. 

Cloudron is great option to install chatwoot including all changes in the github repository automatically cloudron apply to the code.
